### ISSUE TITLE
List feature doesn't work with manual install

### ISSUE THREAD

**Issue by NightMean** [deepjyoti30/ytmdl#145]
# Bug report

## Checklist

- [x] I have searched for similar bugs in the issues.
- [x] I will add the bug label to this issue.

## Version Info

- Ytmdl version: `2020.11.20-1`
- Youtube-dl version: `2021.01.03`

## How Ytmdl was installed?

git clone https://github.com/deepjyoti30/ytmdl && cd ytmdl && sudo python setup.py install

## Console Log

```console
 michal@Michal-PC:/mnt/c/Users/Michal/Ubuntu/ytmdl/ytmdl-modified$ ytmdl --list /mnt/d/Music/1Names.txt --level DEBUG
 ==> Logger running in DEBUG mode
 ==> Passed args: Namespace(SONG_NAME=[], album=None, artist=None, choice=None, disable_file=False, disable_metaadd=False, format='mp3', get_opts=False, ignore_errors=False, level='DEBUG', list='/mnt/d/Music/1Names.txt', list_level=False, manual_meta=False, nolocal=False, pl_end=None, pl_items=None, pl_start=None, proxy=None, quiet=False, skip_meta=False, song=None, title_as_name=False, trim=False, url=None)
 ==> Song Name is required. Check 'ytmdl --help' for help.
```

## Description

Hi again :), 
After the fix of #143 I downloaded ytmdl manually through command that you provided. After passing the file with the --list command, ytdml did not recognize the file location. File is properly created with UTF-8 formatting with LF line breaks only.
I managed to replicate this always when ytmdl is installed manually through that command. 
If ytmdl is installed through pip then the same file is recognized correctly and everything works as expected.

**Comment by deepjyoti30** [comment]
@NightMean Thanks for reporting the issue. On the unstable branch, because of the refactor, the `list` was ignored. It should be fixed in 7f78cf26c12d9fa6eda57d019108765ca5c8452d

**Comment by NightMean** [comment]
Thank you! Works as expected now :)
